packages/components/src/components/atoms/BaseCard/BaseCard.tsx

```diff
@@ -1,19 +1,37 @@
 /* eslint-disable max-lines */
-import { CardHandle } from '@models/CardHandle';
-import { CardProps } from '@models/CardProps';
-import { Position } from '@models/Position';
-import classnames from 'classnames';
-import React, { forwardRef, KeyboardEvent, useImperativeHandle, useState, useEffect, useCallback } from 'react';
+import React, { FunctionComponent } from 'react';
+import { Draggable } from '@templates/Draggable';
+import classNames from 'classnames';
 import { createUseStyles } from 'react-jss';
+import { Dimensions } from '@models/Dimensions';
+import { Position } from '@models/Position';
 
 /**
  * Properties for BaseCard component
  */
-export interface BaseCardProps extends CardProps {
+export interface BaseCardProps {
+  /** Indicates if the card is face up or face down */
+  faceUp?: boolean;
+  /** Defines the card height */
+  height?: number;
+  /** Defines the card width */
+  width?: number;
+  /** Disables the events. Use this if you want to control the card soley via props */
+  disableNativeEvents?: boolean;
+  /** Function called when card position changes with the new position of the card */
+  onPositionChanged?: (position: Position) => void;
+  /** Function called when card flips with a boolean representing whether face is up or not */
+  onFlipped?: (isFaceUp: boolean) => void;
+  /** Indicates the position of the card */
+  position?: Position;
+  /** Indicates the boundaries for the card position */
+  boundaries?: Dimensions;
```

Comment:
Basically, we need to pass the boundaries so that the card cannot be dragged to a position outside of the table. They are used in the function `calculatePositionWithinBoundaries` where the position is calculated so that it stays within the limits of its container

Reply:
aaah ok ok - should we rename to something more explicit like `dragBoundaries`? Or on the comments, to explain a little better that?

Reply:
Maybe `positionBoundaries` rather than `dragBoundaries`? and yeah I agree I will improve the comment

Reply:
`containerBoundaries`?